docker/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM python:3.8.5-slim@sha256:9c84459051ee7e9d386b24ee5468352d52b6789a5d4a8cf6a649a8a1c6ad5636 as socorro_image_base
+FROM mozilla/socorro-minidump-stackwalk:2020.08.31@sha256:fa842d7a3fd84c01acceba9360562f57f0be8a59a21f90fa788689e2feab63ea as breakpad
```

Comment:
It's easier to add this as a `FROM` so as to get the tag and sha256. You could do `COPY --from=REF`, but then it's deep in the Dockerfile and harder (in my opinion) to find and update. Plus I have no idea if dependabot and friends know what to do with it there.